docs/BuildingLBANN.md

```diff
@@ -93,10 +100,10 @@ The following LLNL-maintained packages are optional.
     For example if you are a developer and want to build the inside of
     the git repo use the following instructions:[^libarypath]
```

Comment:
Is `[^libarypath]` a part of the instructions?

Reply:
I think that was @bvanessen's ill-fated attempt to add footnotes. It's gone.